browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchFragment.java

```diff
@@ -140,7 +185,7 @@ public void browserSwitch(int requestCode, Intent intent) {
         }
 
         mRequestCode = requestCode;
-        mContext.startActivity(intent);
+        requireActivity().startActivity(intent);
```

Comment:
Note: An alternative would be to get rid of this `mContext` field which is set to the application context. That would also require changes to the `braintree_android` repository, as `BraintreeFragment` also sets `mContext` to the application context.

I didn't know if `mContext` had to be the app context for any particular reason, so I left it this way. Because of this, I had to change the assertions in `BrowserSwitchFragmentTest` to use Robolectric apis to validate the launched intent, instead of mockito's apis on the mocked `Context`.